functions/data_transformations.R

```diff
@@ -0,0 +1,63 @@
+# Transformation of 'stations_metadata' to data frame:
+
+stations_metadata[[1]][[1]] %>%
```

Comment:
This line does not change `stations_metadata[[1]][[1]]` to a tibble. If you wanted to do that you would write the following:
```
stations_metadata[[1]][[1]] <- 
   stations_metadata[[1]][[1]] %>%
     as_tibble()
```